app/src/main/java/com/example/android/roomwordssample/WordRoomDatabase.java

```diff
@@ -34,7 +34,7 @@
 
     public abstract WordDao wordDao();
 
-    private static WordRoomDatabase INSTANCE;
+    private static volatile WordRoomDatabase INSTANCE;
```

Comment:
Add comment as to what "volatile" means.

Reply:
done